components/_twig-components/functions/bem.function.php

```diff
@@ -6,7 +6,7 @@
 
 use Drupal\Core\Template\Attribute;
 
-$function = new Twig_SimpleFunction('bem', function ($context, $base_class, $modifiers = array(), $blockname = '') {
+$function = new Twig_SimpleFunction('bem', function ($context, $base_class, $modifiers = array(), $blockname = '', $extra = array()) {
```

Comment:
Looks like this file is still in the repo even though we're going to gitignore it. That's because git doesn't forget files that were in version control before you ignore them. You need to `git rm --cached <file>` to remove it from the repo.